kotlin/build.gradle

```diff
@@ -17,6 +17,7 @@ buildscript {
   ext.versions = [
       'targetSdk': 29,
       'compose': '0.1.0-dev05',
+      'kotlin': '1.3.70',
```

Comment:
Didn't you previously mention that dependabot doesn't pick these up?

Reply:
Yea, but as mentioned in the commit message it doesn't really matter. dependabot will only upgrade one dep at a time, and so we always have to manually edit the PRs to update everything at once anyway.